### PR TITLE
app-text/xapers: add 0.9.3 and python 3.12

### PR DESCRIPTION
**Pull request by WPettersson**
* Older versions don't support python 3.12

Closes: https://bugs.gentoo.org/929369

### CODE REVIEW

**Comment by WPettersson** [comment]
Just rebased onto latest master, no other changes with the last force push.

**Comment by WPettersson** [comment]
Another rebase onto master today

File: app-text/xapers/xapers-0.9.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Copyright 2016-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```
Comment (juippis) [inline app-text/xapers/xapers-0.9.3.ebuild:4]: Always aim to use the latest available EAPI, 8 works here.

**Review by juippis — APPROVED** [review]
Sorry for the delay and thanks!

**Pull request closed without merging**